end-to-end-demo/debezium-vertx-demo/src/main/java/io/debezium/example/endtoend/vertx/Verticle.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.example.endtoend.vertx;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.reactivex.config.ConfigRetriever;
+import io.vertx.reactivex.core.AbstractVerticle;
+import io.vertx.reactivex.core.buffer.Buffer;
+import io.vertx.reactivex.core.http.ServerWebSocket;
+import io.vertx.reactivex.core.net.SocketAddress;
+import io.vertx.reactivex.kafka.client.consumer.KafkaConsumer;
+
+public class Verticle extends AbstractVerticle {
+
+    private static final String CONF_BOOTSTRAP_SERVERS = "bootstrap.servers";
+    private static final String CONF_TOPIC = "demo.topic";
+    private static final String CONF_PORT = "demo.port";
+
+    private static final Logger LOG = LoggerFactory.getLogger(Verticle.class);
+
+    @Override
+    public void start() throws Exception {
+        final Map<SocketAddress, ServerWebSocket> sockets = new ConcurrentHashMap<>();
+
+        config().put(CONF_BOOTSTRAP_SERVERS, "localhost:9092");
```

Comment:
Indeed localhost? Not kafka?

Reply:
Good catch! I am using an externa config property so I forgot to change the default value.